manifests/bucketeer/charts/web/values.yaml

```diff
@@ -1,24 +1,112 @@
-fullnameOverride: "web-gateway"
+image:
+  repository: ghcr.io/bucketeer-io/bucketeer-web
+  pullPolicy: IfNotPresent
+fullnameOverride: "web"
 namespace: default
 env:
+  cloudService:
+  profile: true
+  bucketeerTestEnabled:
   gcpEnabled: true
+  bigqueryEmulatorHost:
+  pubsubEmulatorHost:
+  project:
+  mysqlUser:
+  mysqlPass:
+  mysqlHost:
+  mysqlPort: 3306
+  mysqlDbName:
+  persistentRedis:
+    serverName:
+    addr:
+    poolMaxIdle: 25
+    poolMaxActive: 25
+  nonPersistentRedis:
+    serverName:
+    addr:
+    poolMaxIdle: 25
+    poolMaxActive: 25
+  bigQueryDataSet:
+  bigQueryDataLocation:
+  domainTopic:
+  bulkSegmentUsersReceivedTopic:
+  accountService: localhost:9001
+  authService: localhost:9001
+  batchService: localhost:9001
+  environmentService: localhost:9001
+  experimentService: localhost:9001
+  featureService: localhost:9001
+  autoOpsService: localhost:9001
+  healthCheckServicePort: 8000
+  accountServicePort: 9091
+  authServicePort: 9092
+  auditLogServicePort: 9093
+  autoOpsServicePort: 9094
+  environmentServicePort: 9095
+  eventCounterServicePort: 9096
+  experimentServicePort: 9097
+  featureServicePort: 9098
+  notificationServicePort: 9100
+  pushServicePort: 9101
+  webConsoleServicePort: 9102
+  metricsPort: 9002
+  timezone: UTC
+  emailFilter:
+  logLevel: info
+  googleTagManagerId:
+affinity: {}
+nodeSelector: {}
+pdb:
+  enabled:
+  maxUnavailable: 50%
+hpa:
+  enabled:
+  minReplicas:
+  maxReplicas:
+  metrics:
+    cpu:
+      targetAverageUtilization:
 tls:
-  bucketeerJP:
-    secrets:
+  service:
+    secret:
     cert:
     key:
-  service:
+  issuer:
     secret:
     cert:
+  bucketeerJP:
+    secrets:
+    cert:
```

Comment:
Same here.

Reply:
This secrets is used by ingress, we should keep it here.